homeassistant/components/cec.py

```diff
@@ -0,0 +1,120 @@
+"""
+CEC platform.
+
+Requires libcec + Python bindings.
+"""
+
+import logging
+import voluptuous as vol
+from homeassistant.const import EVENT_HOMEASSISTANT_START
+import homeassistant.helpers.config_validation as cv
+
+
+_LOGGER = logging.getLogger(__name__)
+_CEC = None
+DOMAIN = 'cec'
+SERVICE_SELECT_DEVICE = 'select_device'
+SERVICE_POWER_ON = 'power_on'
+SERVICE_STANDBY = 'standby'
+CONF_DEVICES = 'devices'
+ATTR_DEVICE = 'device'
+MAX_DEPTH = 4
+
+
+# pylint: disable=unnecessary-lambda
+DEVICE_SCHEMA = vol.Schema({
+    vol.All(cv.positive_int): vol.Any(lambda devices: DEVICE_SCHEMA(devices),
+                                      cv.string)
+})
+
+
+PLATFORM_SCHEMA = vol.Schema({
```

Comment:
You are defining a component, not a platform, so this should be called `CONFIG_SCHEMA` and contain the domain name too. See [MQTT component](https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/mqtt/__init__.py#L87-L107) for example.